src/templates/blog-post.jsx

```diff
@@ -15,8 +17,12 @@ export default function BlogPost({ data }) {
   // highlight-end
   return (
     <Layout>
-      <div dangerouslySetInnerHTML={{ __html: html }} />
-    </Layout>
+      <Link href="#" onClick={() => window.location.href = '/'} color="primary"><Home /></Link>
+      <br />
+      <div style={{ padding: `2rem 2rem` }}>
```

Comment:
Change this to `1.5rem` and delete `<br />`
```suggestion
      <div style={{ padding: `1.5rem 1.5rem` }}>
```

Reply:
@ming-tsai made the changes